app/src/main/java/com/example/android/sunshine/SettingsFragment.java

```diff
@@ -98,6 +98,9 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
             // we've changed the location
             // Wipe out any potential PlacePicker latlng values so that we can use this text entry.
             SunshinePreferences.resetLocationCoordinates(activity);
+
+            /*This call could potentially cause a repeat too, Better to do a check for which units, or zip
+            when initializing service. Going to commet this out for now*/
```

Comment:
Good analysis, although, the comment doesn't do what it says it will do (you didn't actually comment it out - just pointing this out since comments should be consistent with their code)